# Patch to fix Python bindings installation
# Copyright (C) 2015 Georgi D. Sotirov <gdsotirov@dir.bg>
#
# This program is free software; you can redistribute it and/or
# modify it under the terms of the GNU General Public License
# as published by the Free Software Foundation; either version 2
# of the License, or (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
# GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License
# along with this program; if not, write to the Free Software
# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
#
diff -urNad gdal-1.10.1-orig/swig/python/GNUmakefile gdal-1.10.1/swig/python/GNUmakefile
--- gdal-1.10.1-orig/swig/python/GNUmakefile	2013-08-26 23:01:11.000000000 +0300
+++ gdal-1.10.1/swig/python/GNUmakefile	2015-04-26 18:03:48.000000000 +0300
@@ -68,9 +68,9 @@
 install:
 
 ifeq ($(PY_HAVE_SETUPTOOLS),1)
-	$(PYTHON) setup.py install 
+	$(PYTHON) setup.py install --root=${DESTDIR}
 else
-	$(PYTHON) setup.py install --prefix=$(DESTDIR)$(prefix)
+	$(PYTHON) setup.py install --root=${DESTDIR} --prefix=$(prefix)
 endif
 
 	for f in $(SCRIPTS) ; do $(INSTALL) ./scripts/$$f $(DESTDIR)$(INST_BIN) ; done
